Gemfile.lock

```diff
@@ -9,6 +9,11 @@ GEM
   remote: https://rubygems.org/
   specs:
     ZenTest (4.11.1)
+    activesupport (5.2.3)
```

Comment:
note that this gem is going to be used in the wild, so the less dependencies the better. This at the moment would for example tie this gem to Rails 5.2.

Also note as far as I know Gemfile.lock should not be commited for public gems, only the Gemfile itself

Reply:
Adding to this, your build will fail unless you remove the lockfile. Your Gemfile specifies ZenTest 4.11.2 which is necessary to pass Travis, but the lockfile sets ZenTest to 4.11.1. It's an issue that  [this OpenApi release](https://github.com/OpenAPITools/openapi-generator/releases/tag/v4.0.0-beta3) will fix, but for now make sure v4.11.2 is specified.